sdk/cognitivelanguage/azure-ai-language-conversations/setup.py

```diff
@@ -75,8 +74,8 @@
         'azure.ai.language.conversations': ['py.typed'],
     },
     install_requires=[
-        "azure-core<2.0.0,>=1.28.0",
-        "isodate<1.0.0,>=0.6.1",
+        "azure-core>=1.28.0",
+        "isodate>=0.6.1",
```

Comment:
should we keep upper bound for packages we don't own?

Reply:
No. We don't want the upper bound.

We cannot stop customers from using the latest one which may contain security patch.